runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java

```diff
@@ -71,6 +71,8 @@
 	@Inject
 	private RuneLiteProperties runeLiteProperties;
 
+	private final Pattern SPLIT_REGEX = Pattern.compile("\\s*,\\s*");
```

Comment:
also this should be static final

Reply:
oh and also above fields

Reply:
`public static final Splitter SPLITTER = Splitter.on(",").trimResults();`